core/gui/src/app/dashboard/component/user/google-login/google-login.component.ts

```diff
@@ -25,8 +25,11 @@ export class GoogleLoginComponent implements AfterViewInit {
     this.googleAuthService.googleCredentialResponse
       .pipe(
         mergeMap(res => this.userService.googleLogin(res.credential)),
-        untilDestroyed(this)
+        take(1)
```

Comment:
Why do you need to replace untilDestroyed(this) with take(1)?

Reply:
Because if `untilDestroyed(this)` is used, the component might cancel the subscription too early, preventing it from entering `subscribe`. However, to ensure automatic unsubscription, I used `take(1)` to let it automatically unsubscribe after the first event is triggered.